### DISCUSSION TITLE
Channel Test

### DISCUSSION THREAD

**Discussion by Sphinxroot** [iptv-org/community#267] [Show and tell]
DE  GEO LOCKED

#EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 1
http://wdrardevent1-lh.akamaihd.net/i/ardevent1_weltweit@566648/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 2
https://ardevent2.akamaized.net/hls/live/681512/ardevent2_geo/master.m3u8

#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 1
http://ndrspezial-lh.akamaihd.net/i/spezial_1@430235/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 2
http://ndrspezial-lh.akamaihd.net/i/spezial_2@430236/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 3
http://ndrspezial-lh.akamaihd.net/i/spezial_3@430237/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 4
http://ndrspezial-lh.akamaihd.net/i/spezial_4@430238/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 5
http://ndrspezial-lh.akamaihd.net/i/spezial_5@430239/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 6
https://ndrspezial-lh.akamaihd.net/i/spezial_6@430240/master.m3u8

#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 7
https://zdf-hls-07.akamaized.net/hls/live/2016302/de/veryhigh/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 8
https://zdf-hls-08.akamaized.net/hls/live/2016303/de/veryhigh/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 9
https://zdf-hls-09.akamaized.net/hls/live/2016304/de/veryhigh/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 10
https://zdf-hls-10.akamaized.net/hls/live/2016305/de/veryhigh/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 11
https://zdf-hls-11.akamaized.net/hls/live/2016436/de/veryhigh/master.m3u8
#EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 12
https://zdf-hls-12.akamaized.net/hls/live/2016495/de/high/master.m3u8

#EXTINF:-1,?
https://zdf-hls-11.akamaized.net/hls/live/2016494/de/high/master.m3u8
#EXTINF:-1,?
https://zdf-hls-13.akamaized.net/hls/live/2016496/de/high/master.m3u8
#EXTINF:-1,?
https://zdf-hls-14.akamaized.net/hls/live/2016497/de/high/master.m3u8

We  Have Those In list
#EXTINF:-1,ZDF
https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8
#EXTINF:-1,ZDFneo
https://zdf-hls-16.akamaized.net/hls/live/2016499/de/high/master.m3u8
#EXTINF:-1,ZDFinfo
https://zdf-hls-17.akamaized.net/hls/live/2016500/de/high/master.m3u8
#EXTINF:-1,3sat
https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/high/master.m3u8
#EXTINF:-1,Phoenix
https://zdf-hls-19.akamaized.net/hls/live/2016502/de/high/master.m3u8

**Comment by jkemp814** [comment]
The first two, and 15 on to 19 play from Arizona right inside Firefox tabs

**Comment by BellezaEmporium** [comment]
> 
> 
> #EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 1 (works on FR, US)
> http://wdrardevent1-lh.akamaihd.net/i/ardevent1_weltweit@566648/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 2 (geolocked on FR)
> https://ardevent2.akamaized.net/hls/live/681512/ardevent2_geo/master.m3u8
> 
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 1 (geolocked on FR)
> http://ndrspezial-lh.akamaihd.net/i/spezial_1@430235/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 2 (geolocked on FR)
> http://ndrspezial-lh.akamaihd.net/i/spezial_2@430236/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 3 (geolocked on FR)
> http://ndrspezial-lh.akamaihd.net/i/spezial_3@430237/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 4 (geolocked on FR)
> http://ndrspezial-lh.akamaihd.net/i/spezial_4@430238/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 5 (geolocked on FR)
> http://ndrspezial-lh.akamaihd.net/i/spezial_5@430239/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/ZQXWoHC.png" group-title="",NDR Event Olympia 6 (geolocked on FR)
> https://ndrspezial-lh.akamaihd.net/i/spezial_6@430240/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 7 (geolocked on FR)
> https://zdf-hls-07.akamaized.net/hls/live/2016302/de/veryhigh/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 8 (geolocked on FR)
> https://zdf-hls-08.akamaized.net/hls/live/2016303/de/veryhigh/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 9 (geolocked on FR)
> https://zdf-hls-09.akamaized.net/hls/live/2016304/de/veryhigh/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 10 (geolocked on FR)
> https://zdf-hls-10.akamaized.net/hls/live/2016305/de/veryhigh/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 11 (geolocked on FR)
> https://zdf-hls-11.akamaized.net/hls/live/2016436/de/veryhigh/master.m3u8
> #EXTINF:-1 tvg-logo="https://i.imgur.com/aDtEi8D.png" group-title="",ZDF Event Olympia 12 (geolocked on FR)
> https://zdf-hls-12.akamaized.net/hls/live/2016495/de/high/master.m3u8
> 
> #EXTINF:-1,? (geolocked on FR)
> https://zdf-hls-11.akamaized.net/hls/live/2016494/de/high/master.m3u8
> #EXTINF:-1,? (geolocked on FR)
> https://zdf-hls-13.akamaized.net/hls/live/2016496/de/high/master.m3u8
> #EXTINF:-1,? (geolocked on FR)
> https://zdf-hls-14.akamaized.net/hls/live/2016497/de/high/master.m3u8
> 
> We Have Those In list
> #EXTINF:-1,ZDF (geolocked on FR)
> https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8
> #EXTINF:-1,ZDFneo (geolocked on FR)
> https://zdf-hls-16.akamaized.net/hls/live/2016499/de/high/master.m3u8
> #EXTINF:-1,ZDFinfo (geolocked on FR)
> https://zdf-hls-17.akamaized.net/hls/live/2016500/de/high/master.m3u8
> #EXTINF:-1,3sat (geolocked on FR)
> https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/high/master.m3u8
> #EXTINF:-1,Phoenix (geolocked on FR)
> https://zdf-hls-19.akamaized.net/hls/live/2016502/de/high/master.m3u8

**Comment by jkemp814** [comment]
#EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 2 (geolocked on FR)
https://ardevent2.akamaized.net/hls/live/681512/ardevent2_geo/master.m3u8

That one actually plays. Even though it's marked [GEO-Blocked]

These also play. From US.

#EXTINF:-1,ZDF (geolocked on FR)
https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8
#EXTINF:-1,ZDFneo (geolocked on FR)
https://zdf-hls-16.akamaized.net/hls/live/2016499/de/high/master.m3u8
#EXTINF:-1,ZDFinfo (geolocked on FR)
https://zdf-hls-17.akamaized.net/hls/live/2016500/de/high/master.m3u8
#EXTINF:-1,3sat (geolocked on FR)
https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/high/master.m3u8
#EXTINF:-1,Phoenix (geolocked on FR)
https://zdf-hls-19.akamaized.net/hls/live/2016502/de/high/master.m3u8

**Comment by BellezaEmporium** [comment]
> 
> 
> #EXTINF:-1 tvg-logo="https://i.imgur.com/luroyTG.png" group-title="",ARD Event Olympia 2 (geolocked on FR)
> https://ardevent2.akamaized.net/hls/live/681512/ardevent2_geo/master.m3u8
> 
> That one actually plays. Even though it's marked [GEO-Blocked]
> 
> These also play. From US.
> 
> #EXTINF:-1,ZDF (geolocked on FR)
> https://zdf-hls-15.akamaized.net/hls/live/2016498/de/high/master.m3u8
> #EXTINF:-1,ZDFneo (geolocked on FR)
> https://zdf-hls-16.akamaized.net/hls/live/2016499/de/high/master.m3u8
> #EXTINF:-1,ZDFinfo (geolocked on FR)
> https://zdf-hls-17.akamaized.net/hls/live/2016500/de/high/master.m3u8
> #EXTINF:-1,3sat (geolocked on FR)
> https://zdf-hls-18.akamaized.net/hls/live/2016501/dach/high/master.m3u8
> #EXTINF:-1,Phoenix (geolocked on FR)
> https://zdf-hls-19.akamaized.net/hls/live/2016502/de/high/master.m3u8

First one streams nothing but a 20 second video saying that this content isn't available outside of Germany, and the others get me an Access Denied.

![vlcsnap-2021-07-24-21h54m44s760](https://user-images.githubusercontent.com/30985701/126879789-88ddef16-8bcf-4f0e-858c-2c5f1a8286d5.png)

**Comment by jkemp814** [comment]
I have seen that on a couple of them. The others are playing different content like westerns and movies. Seems like a talk show on others. A music channels.

**Comment by jkemp814** [comment]
![zdf-hls-19](https://user-images.githubusercontent.com/12059343/126880291-394577e7-b748-4eaa-ac86-cd0f01c0915e.jpg)
![zdf-hls-15](https://user-images.githubusercontent.com/12059343/126880026-6a095271-bfe8-4d36-b43f-e9adeb8efed0.jpg)
![zdf-hls-16](https://user-images.githubusercontent.com/12059343/126880031-cdebf513-6877-4699-b9ad-dac351165a50.jpg)
![zdf-hls-17](https://user-images.githubusercontent.com/12059343/126880034-9f34ad69-6d8e-4d0b-9916-60d1950a0bc6.jpg)
![zdf-hls-18](https://user-images.githubusercontent.com/12059343/126880037-fbcabca8-5f7f-4908-a273-855246f69339.jpg)

**Comment by jkemp814** [comment]
Was just trying to point out they are playing even though they are flagged as geo-blocked

I'm playing them right inside of Firefox tab, not VLC.

**Comment by BellezaEmporium** [comment]
Weird... Firefox doesn't play streams for me.

**Comment by jkemp814** [comment]
There is an extension you have to add, one moment.

https://addons.mozilla.org/en-US/firefox/addon/native-mpeg-dash-hls-playback/?utm_source=addons.mozilla.org&utm_medium=referral&utm_content=search

I right-click the link above and open in new tab

Sometimes it may complain the browser is already open. So when I start my code editing, I have my browser close initially. Then it will play the links from my editor, so I can stream and test the channel.

**Comment by BellezaEmporium** [comment]
I've tried, but it doesn't work either.

**Comment by jkemp814** [comment]
Hmmm, works like a charm on my end. Not sure what could be wrong. I can click on a link like in this thread, or from my editor (follow-link), and it just works.

**Comment by Food4Monkeys** [comment]
Sphinxroot, du hast kein Benehmen und keinen Anstand!!! In der Open Source Szene gibt man auch die Quelle an, wenn man schon die Arbeit anderer kopiert!!! 

https://github.com/Free-IPTV/Countries/commit/55c03fb35356d010cc5d76978714fb7f5630a4f0#commitcomment-53941754

**Comment by Sphinxroot** [comment]
> Sphinxroot, du hast kein Benehmen und keinen Anstand!!! In der Open Source Szene gibt man auch die Quelle an, wenn man schon die Arbeit anderer kopiert!!!
> 
> [Free-IPTV/Countries@55c03fb#commitcomment-53941754](https://github.com/Free-IPTV/Countries/commit/55c03fb35356d010cc5d76978714fb7f5630a4f0#commitcomment-53941754)

Oh My !   Here We Go Again !!

first : ive found it on a t.me groups .. big deal , ive already said that stuff get everywhere and quick .. unfortunately, you guys definitively  don`t understand that ..

second : I don`t need your repo to find stuff , I did`nt nocticed , that you where  back online , since you started scamming peoples 3 euro for "free and other peoples work" in a private repo 
i ignored you .. but ,as i see ,you still making problem over a picture in link .. you guys are funny and entertaining , you should seel ticket for the show .. il buy the first 1 , and hope i get a dedication on it !

third : i also saw that you , started a vendenta (not to mention  who , The list is to long ), over the top guys of the free open source iptv community , you just makin a fool of yourself realy .. peoples know them well , and they are highly respected , you are loosing ground
if you think , you stand a chance . your just wasting your time over something that you can`t control ..

in last : Just built you community on your side and don`t bother with other .. where stuff go ,and so on ,... start to be generous  .. peoples will join in and in the long term it will be good for you 

best regard

**Comment by Food4Monkeys** [comment]
@Sphinxroot,

was du hier schreibst, ist reiner Dünnschiss! Ich habe keine Repo mit IPTV, ich habe lediglich ein Fork von Free-IPTV! Anscheinend bist du auch dazu zu blöd, das zu sehen aber ein Copy & Paste, das kannst du sehr gut aber das kann auch jeder DAU!  

Du bist einfach nur eine Schande für die Open Source Community. weil du die Regeln missachtest.! Lass einfach dein dummes kopieren von anderen, OHNE die Quelle anzugeben.Dein Verhalten ist einfach nur armseelig!
